### PR TITLE
BUG/DOC/TST: fixes #625

### LINKED ISSUE #625: Table.nonzero() only works if the table is in csr format
The transform [`tocsr`](https://github.com/biocore/biom-format/blob/master/biom/table.py#L2739) is not inplace, but the code assumes it is. Therefore, the code only works if the table is already in csr.

### PR DESCRIPTION
**Pull request by wasade**
As the title states.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Bug fixes:
 * Improve error handling when filtering tables, and return tables of shape
     `(0, n)` instead of `(0, 0)` when fully filtering out a table along an
     axis, issue #620
+* Fix `Table.nonzero` to work on data is not already csr, issue #625.
```
Comment (Jorge-C) [inline ChangeLog.md]: Maybe add a `that` before `is`?

**Comment by Jorge-C** [comment]
Just a question about English, otherwise :+1:

**Comment by wasade** [comment]
Fixed, thanks @Jorge-C!

**Comment by Jorge-C** [comment]
Thanks Daniel!

**Pull request merged**